src/float32sr.jl

```diff
@@ -53,6 +54,10 @@ Base.Float64(x::Float32sr) = Float64(Float32(x))
 Float32sr(x::Integer) = Float32sr(Float32(x))
 (::Type{T})(x::Float32sr) where {T<:Integer} = T(Float32(x))
 
+# converting to and from BFloat16
+Float32sr(x::BFloat16) = Float32sr(Float64(x))
```

Comment:
This should go via `Float32` and not `Float64`? That will just padd trailing zeros and reinterpret